config/defaults.go

```diff
@@ -19,8 +19,9 @@ var DefaultNodeConfig = NodeConfig{
 	Aggregator: true,
 	BlockManagerConfig: BlockManagerConfig{
 		BlockTime:           200 * time.Millisecond,
+		EmptyBlocksMaxTime:  0,
```

Comment:
IMO should be enabled by default. Actually don't think it should even be a flag. thinking it should simply not produce empty blocks hardcoded but maybe I'm missing something. 

is there any reason you think an operator would like to produce empty blocks assuming no cron job depends on it?

Reply:
In case some modules have begin block / end block logic that needs to run even on empty blocks
(e.g epoch, which checks the timers and call the hooks on begin block)